LoopKitUI/CGMManagerUI.swift

```diff
@@ -10,19 +10,27 @@ import SwiftUI
 import LoopKit
 
 public protocol CGMManagerUI: CGMManager, DeviceManagerUI, PreferredGlucoseUnitObserver {
-    /// Provides a view controller for setting up and configuring the manager if needed.
+    /// Create and onboard a new CGM manager.
     ///
-    /// If this method returns nil, it's expected that `init?(rawState: [:])` creates a non-nil manager
-    static func setupViewController(glucoseTintColor: Color, guidanceColors: GuidanceColors) -> (UIViewController & CGMManagerSetupViewController & CompletionNotifying)?
+    /// - Parameters:
+    ///     - colorPalette: Color palette to use for any UI.
+    /// - Returns: Either a conforming view controller to create and onboard the CGM manager, a newly created and onboarded CGM manager, or an error.
+    static func setupViewController(colorPalette: LoopUIColorPalette) -> UIResult<UIViewController & CGMManagerCreateNotifying & CGMManagerOnboardNotifying & CompletionNotifying, CGMManagerUI, Error>
 
-    func settingsViewController(for glucoseUnit: HKUnit, glucoseTintColor: Color, guidanceColors: GuidanceColors) -> (UIViewController & CompletionNotifying & PreferredGlucoseUnitObserver)
+    /// Configure settings for an existing CGM manager.
+    ///
+    /// - Parameters:
+    ///     - glucoseUnit: The glucose units to use.
```

Comment:
[nit] I have been updating `glucoseUnit` to `preferredGlucoseUnit` when it has just been for display purposes. At least in the comment considering clarifying that the use is for display purposes.

Reply:
Ah, will do.